Formula/lmod.rb

```diff
@@ -1,8 +1,8 @@
 class Lmod < Formula
   desc "Lua-based environment modules system to modify PATH variable"
-  homepage "https://www.tacc.utexas.edu/research-development/tacc-projects/lmod"
-  url "https://github.com/TACC/Lmod/archive/8.4.1.tar.gz"
-  sha256 "f1072e732d2690dd542935542147fb1c3f31c79b73b8ffdffff1c0a0ffa90737"
+  homepage "https://lmod.readthedocs.io"
```

Comment:
This looks like documentation instead of a homepage

Reply:
This is actually the page featured in the repo. Yeah, the utexas one does still have cert issue though.